Src/LiquidProjections.NEventStore/packages.config

```diff
@@ -1,6 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="LibLog" version="4.2.6" targetFramework="net452" developmentDependency="true" />
-  <package id="LiquidProjections.Abstractions" version="2.0.0" targetFramework="net45" />
-  <package id="NEventStore" version="5.0.1.2" targetFramework="net45" />
+  <package id="LiquidProjections.Abstractions" version="2.2.0" targetFramework="net45" />
+  <package id="LiquidProjections.PollingEventStore.Sources" version="1.0.1" targetFramework="net45" />
+  <package id="Microsoft.CSharp" version="4.3.0" targetFramework="net45" />
+  <package id="NEventStore" version="5.2.0" targetFramework="net45" />

```

Comment:
Is it only compatible with NEventStore 5.2.0 now? We're still using NEventStore 5.0.1.2 in Flow Vision 7.